front-end/src/components/HeaderLogo.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import logo from '../images/logo.png'
+import logo from '../images/newLogo.png'
```

Comment:
newLogo seems to be an old version of the new logo; i'd suggest sticking with logo.png